File: minions/13.El medio hot-dog/Elmedio hot-dog.py
N = int(input())

total = int(input().split(".")[1])

t = 0

for i in range(N):
    deposito = int(input().split("."[1]))
    total += deposito
    
    if total % 100 == 0:
        total=0
    else:
        t += 1

print(t)
# no puedo generar los casos

File: minions/13.El medio hot-dog/Elmedio_hot-dog.c
#include <stdio.h>
#include <stdlib.h>

int main(){
    int N, i, t = 0;
    float deposito, total;

    do{
        scanf("%d", &N);
    } while (N <= 0);

    do{
        scanf("%f", &total);
    } while (total <= 0);
    
    for (i = 0; i < N; i++){
        scanf("%f", &deposito);
        total += deposito;

        if ((int)total / total != 1)
            t++;
        else
            total = 0;
    }
    
    printf("%d", t);

    return EXIT_SUCCESS;
}